src/app/core/utility-module/utility.service.ts

```diff
@@ -0,0 +1,73 @@
+import { HttpClient, HttpErrorResponse } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { UTILITY_ROUTES } from './utility.routes';
+import { AuthService } from '../auth-module/auth.service';
+import { Observable, throwError } from 'rxjs';
+import { Blog } from 'app/components/blogs/types/blog';
+import { catchError, retry } from 'rxjs/operators';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UtilityService {
+  constructor(private http: HttpClient, private auth: AuthService) { }
+
+  /**
+   * Gets all blogs from the database
+   *
+   * @returns An observable containing an array of blogs
+   */
+  getAllBlogs(): Observable<Blog[]> {
+    return this.http.get<Blog[]>(UTILITY_ROUTES.GET_BLOGS());
+  }
+
+  // /**
```

Comment:
Is the create blog used?

Reply:
No, there's not UI. We hope to add this to the client in the future.